settings/OpenExceptionalForm/ExceptionWrapper.js

```diff
@@ -36,36 +38,7 @@ class ExceptionWrapper extends React.Component {
 
   constructor() {
     super();
```

Comment:
You can avoid using constructor in this case if there only state initilization. Just crease state via class property syntax

Reply:
Here is interesting case, in constructor is using setState method (what is not recommended do in constructor, but any attempts to replace it with state initialization and do setState in componentDidMount method bring incorrect work of checkboxes). I suggest to leave constructor method in class.

Reply:
Ok, but pass constuctor argument called props to super